### ISSUE TITLE
Discord Notification

### ISSUE THREAD

**Issue by BassHous3** [Sonarr/Sonarr#7745]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

Using latest version of Sonarr and webhooked to Discord.
Currently, when a download is fetched without a release group it doesn't report this in the notification. It simply leaves it out. Personally, this seems to be like a bug.

![Image](https://github.com/user-attachments/assets/99b45e62-52e0-4e96-8b2c-06657c916389)

### Expected Behavior

It should report as `Group: N/A` or  `Group: -` or `Group not found` 

Suggested example:
![Image](https://github.com/user-attachments/assets/4d32e13f-0c5d-4bb5-b370-e4a891d732da)

### Steps To Reproduce

1. Hook Sonarr to Discord channel
2. Fetch a download without a group
3. Read the notification sent to Discord

### Environment

```markdown
- OS: Linux Server 24.10
- Sonarr: 4.0.14.2939
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: Brave - Version 1.76.80
- Database: Sqlite 3.48.0
```

### What branch are you running?

Main

### Trace Logs?

N/A

**Comment by BassHous3** [comment]
The best way to look at it is:
If everything is missing, then you would know that the fetched release is probably bad and you have to act on it. 
Showing a missing group has a similar approach, chances that the release is unreliable are higher. 

Right now it looks like it wasn't able to read the information rather than reporting it was missing. 

That is my take on it, at least. Although I certainly understand your point as well.

**Comment by markus101** [comment]
I can see why someone might want to see placeholders instead of nothing, but that's not the approach we took. If someone is interested in implementing an option for that, great, but we're not going to commit to doing that at this time.
